gridgencore.py

```diff
@@ -8,7 +8,7 @@
 
 ASSET_DIR = os.path.dirname(__file__) + "/assets"
 EXTRA_FOOTER = "..."
-EXTRA_ASSETS = []
+EXTRA_ASSETS = ["placeholder-landscape.png", "placeholder-portrait.png", "placeholder-square.png"]
```

Comment:
This should not be here, it should be on line 583 (same place you added `styles.css`)